index.html

```diff
@@ -1,21 +1,109 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="css/header.css">
+    <link rel="stylesheet" href="css/main.css">
+    <link rel="stylesheet" href="css/footer.css">
+    <link rel="stylesheet" href="css/common.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@600;700&family=Quicksand:wght@300;400;700&display=swap" rel="stylesheet">
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    <header class="header">
+        <div class="header__col1">
+            <div class="header__col1--item1">
+                <div class="header__mobile">
+                    <div class="header__square--mobile"></div>
+                    <div class="header__name--mobile">Maya Nelson<br>
+                        <h3 class="header__role--mobile">PROJECT MANAGER</h3>
+                    </div>
+                </div>
+
+                <div class="header__desktop">
+                    <div class="header__square--desktop"></div>
+                    <div class="header__name--desktop">Maya Nelson</div>
+                    <h3 class="header__role--desktop"> &nbsp; / PROJECT MANAGER</h3>
+                </div>
+            </div>
+        </div>
+        </div>
+
+        <div class="header__col2">
+            <nav class="header__nav">
+                <img src="imgs/ic_hamburger.png" alt="hamburger menu icon" class="ic_hamburger">
+                <a href="#" class="header__nav--link-on">ABOUT ME</a>
+                <a href="#" class="header__nav--link">RESUME</a>
+                <a href="#" class="header__nav--link">PROJECTS</a>
+                <a href="#" class="header__nav--link">PROJECTS</a>
+            </nav>
+        </div>
+    </header>
+
+    <div class="main__grid">
+        <div class="profile">
+            <section class="profile__card">
+                <img src="./imgs/profile.webp" alt="profile image" class="profile__card--img">
```

Comment:
really nice to see you using `webp`. This is a well optimised image format for web!